authentication/esignet-integration-impl/src/main/java/io/mosip/authentication/esignet/integration/service/IdaVCIssuancePluginImpl.java

```diff
@@ -198,7 +198,7 @@ private byte[] b64Decode(String value) {
 
 	//Converts an array of two-letter language codes to their corresponding ISO 639-2/T language codes.
 	private List<String> convertLangCodesToISO3LanguageCodes(String[] langCodes) {
-		if(langCodes == null || langCodes.length == 0)
+		if(langCodes == null || langCodes.length == 0 || (langCodes.length == 1 && langCodes[0].isEmpty()))
```

Comment:
language codes is user input as comma-separated values and then it is split based on comma. Now if the input is ",,,,". then the method returns an empty list. which means the default language as english is not always true.

Reply:
For language codes we are using space as the seperator not comma @ase-101